### ISSUE TITLE
Training and testing data structure

### ISSUE THREAD

**Issue by Opdoop** [TangJiaLong/PSSAttention#2]
The data in this repo is supposed to be extracted from SemEval14. It seems all the data in the repo have an opinion term, but some data in SemEval14 don't have opinion terms. How did you solve this problem?  (If you just ignore the record don't have opinion term, is it still fair to compare with other methods?)

**Comment by Opdoop** [comment]
And in terms of validation, from the data folder, it seems like you split some training data to the validation set in [14semeval_laptop_val ](https://github.com/TangJiaLong/PSSAttention/tree/master/TNet(%2BAS)/dataset/14semeval_laptop_val)and [14semeval_rest_val](https://github.com/TangJiaLong/PSSAttention/tree/master/TNet(%2BAS)/dataset/14semeval_rest_val).
But in the [main_total.py](https://github.com/TangJiaLong/PSSAttention/blob/master/TNet(%2BAS)/main_total.py), no matter which data folder you pointed to, you only train one epoch followed a test, and report the best result in all of this epoch. It seems you didn't validate the model at all. Could you give some feedback? Thanks in advance.

https://github.com/TangJiaLong/PSSAttention/blob/85bd5e8a24675ad845ecb88abd17ab7f5554e0f7/TNet(%2BAS)/main_total.py#L30-L79

**Comment by venusafroid** [comment]
Hi, do you already have your answer? Can you tell me, please
